src/screens/SignedChatScreen/index.tsx

```diff
@@ -73,12 +77,6 @@ const SignedChatScreen = () => {
     }
   }, [selectedChannel]);
 
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The call to `clearReplyPreview` has been removed from the `useEffect` hook. If this function was meant to clear any reply previews when the component unmounts, its removal could lead to undesired behavior. Please ensure that this change doesn't introduce any side effects.



<!-- This is an auto-generated comment by OSS CodeRabbit -->

Reply:
clear